gef-ui/package.json

```diff
@@ -51,7 +51,7 @@
     "date-fns": "^2.30.0",
     "date-fns-tz": "^2.0.1",
     "dotenv": "16.0.3",
-    "express": "^4.21.1",
+    "express": "^4.21.2",
```

Comment:
if we can't upgrade this to the latest version (if breaking), we could use `~`?

Reply:
They are not breaking changes, axios is.